app/__main__.py

```diff
@@ -60,6 +61,7 @@ def get_routes():
             UIStaticHandler,
             dict(path=f"{os.path.dirname(__file__)}/public/static/"),
         ),
+        (r"/admin/scene/([0-9]{0,16})", UIAdminSceneHandler),
```

Comment:
Does `/admin/([a-zA-Z_-]{1,50})/([0-9]{0,16})` make more sense here?

Reply:
Hmmm, since we don't use the scenario friendly name anywhere in this handler i thought it would make sense to leave it out.

Your way also makes sense in a way since that matches the user endpoint.

Should I change to what you have?

Reply:
Wait for Jay's feedback first! It was just an idea